tests/containers/prepare_rootless_podman.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This is the preparation of rootless_podman
+#  * It switches to CGroup v2
+#  * It adds the uset to systemd-journal group
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    assert_script_run "usermod -a -G systemd-journal $testapi::username";
+
+    assert_script_run "sed -i '/^GRUB_CMDLINE_LINUX_DEFAULT/ s/\"\$/ systemd.unified_cgroup_hierarchy=1 \"/' /etc/default/grub";
+    assert_script_run "cat /etc/default/grub";
+    assert_script_run "grub2-mkconfig -o /boot/grub2/grub.cfg";
+
+    select_console 'root-console';
+    enter_cmd "shutdown -r now";
```

Comment:
don't we have `power_action` method to handle reboots?
Btw, this might also impact SLE Micro and MicroOS, where reboot is handled differently.